torch/nested/_internal/ops.py

```diff
@@ -652,9 +652,19 @@ def copy_default(func, *args, **kwargs):
     inp = new_kwargs.pop("input")
     src = new_kwargs.pop("src")
     if inp._size != src._size:
-        raise RuntimeError(
-            "copy_ only supports Nested Tensors that have same size and the exact same offset tensor."
-        )
+        # try to recursively copy_ on unbound components
```

Comment:
Cool way to get around the nested int mismatch! 
Should we add a TODO to say that we'd eventually want to directly copy?